airflow/contrib/sensors/DataLake_PrefixFile_Sensor.py

```diff
@@ -0,0 +1,54 @@
+from airflow.contrib.hooks.azure_data_lake_hook import AzureDataLakeHook
+from airflow.sensors.base_sensor_operator import BaseSensorOperator
+from airflow.utils.decorators import apply_defaults
+
+
+# File detector by prefix sensor for azure data lake storage
+
+class DataLakePrefixSensor(BaseSensorOperator):
+
+    """
+    Interacts with Azure Data Lake:
+
+    Client ID and client secret should be in user and password parameters.
+    Tenant and account name should be extra field as
+    {"tenant": "<TENANT>", "account_name": "ACCOUNT_NAME"}.
```

Comment:
This description doesn't quite describe the purpose of the sensor. Would you mind updating and elaborating here?